src/web/handlers.rs

```diff
@@ -540,6 +540,53 @@ pub async fn heartbeat(hb: HeartbeatRequest) -> Result<HttpResponse, Error> {
     }
 }
 
+pub async fn lbheartbeat(req: HttpRequest) -> Result<HttpResponse, Error> {
+    let mut resp: HashMap<String, Value> = HashMap::new();
+
+    let state = match req.app_data::<Data<ServerState>>() {
+        Some(s) => s,
+        None => {
+            error!("⚠️ Could not load the app state");
+            return Ok(HttpResponse::InternalServerError().body(""));
+        }
+    };
+
+    let deadarc = state.deadman.clone();
+    let mut deadman = *deadarc.read().await;
```

Comment:
This confused me for a while (getting a `mut` here) -- finally realized it's a `Copy` type